CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
+### Added
+* Added `replicaTableLocation`, `replicaMetastoreUris` and  `partitionKeys` to SNS message.
 ### Changed
+* SNS Message `protocolVersion` changed from "1.0" to "2.0.0".
```

Comment:
I don't think we need an extra number for the versions here, M.m should be sufficient

Reply:
But then we lose the benefits of semantic versioning and that you can just point people to the website and say "go there, this explains how our versions work". I'm all for "sem ver everywhere".

Reply:
This is a "protocol", there is no patch/build version in protocols

Reply:
You could make a backwards compatible change to the message that fixes a bug which would result in updating the "hotfix" number. This would be different to making a backwards compatible change to the message that adds a new feature which would result in updating the "minor version" number. So I think you can argue that SemVer is just as useful for API versioning as it is for software versioning.

Reply:
I disagree, any changes in the format make a new version of the protocol no matter if they are backwards compatible or not

Reply:
So you're disagreeing with the definition of SemVer then and suggesting we don't use it here? I'd prefer to use it as its a well agreed upon, documented and known versioning scheme that we can point to instead of having to explain our own here. Any changes do require a new version, it just determines which of the 3 numbers you change and by having 3 instead of 2 you can communicate the difference between a change due to a bug fix and a change due to the addition of a new feature.

Reply:
I disagree with having 3 number for protocol versioning

Reply:
OK, anyone else have strong opinions on this? If necessary I'll set up a quick meeting to get a group decision.

Reply:
I quite like have the same SemVer versioning scheme in all our version numbers it makes it easier to talk about. I don't really see any harm in using it here.